applications/OptimizationApplication/custom_python/optimization_python_application.cpp

```diff
@@ -173,6 +173,7 @@ PYBIND11_MODULE(KratosOptimizationApplication, m)
     //adjoint RHS
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, ADJOINT_RHS);
 
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, SHAPE);
```

Comment:
COORDS ??? as a better name
```suggestion
    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, COORDS);
```

Reply:
Let's keep the old name shape as we used to it and a lot of var in the code follow the name SHAPE. For other shape param, like NURBS, we can use NURBS_SHAPE or similar.